### PR TITLE
181/handle sshkeys

### LINKED ISSUE #181: Handle sshkey
- Change the endpoint to users.
- Change the description for connect Digitalocean

### PR DESCRIPTION
**Pull request by iLeonelPerea**
closes #181 

changes proposed in the pull request:

- Change sshkeys uri
- Handle sshkeys
- Update tests

@thetonymaster

### CODE REVIEW

File: src/components/provisionerForm/SSHKeys.js [hunk 1]
```diff
@@ -13,18 +13,17 @@ const SSHKeys = ( {deleteSSHKey, enableSSHKey, showSSHKey, setSSHKey, setSSHKeyT
   };
   
   const StoreSSHKeyKeypress = (e) => {
-    e.preventDefault();
+    // e.preventDefault();
```
Comment (thetonymaster) [inline src/components/provisionerForm/SSHKeys.js]: Remove this line if it is not used

File: src/containers/ProvisionerForm.js [hunk 2]
```diff
@@ -32,7 +32,7 @@ export class ServiceForm extends Component {
         <div className="row">
           <div className="large-10 columns">
             <h1><i className="step fi-clipboard-notes" />
-              Create a Service
+              Create a Services
```
Comment (thetonymaster) [inline src/containers/ProvisionerForm.js]: Create Services if plural
Create a Service if singular

Comment (iLeonelPerea) [inline src/containers/ProvisionerForm.js]: haha.. I forgot remove that haha

**Comment by thetonymaster** [comment]
LGTM

**Pull request merged**